crates/builder/src/signer.rs

```diff
@@ -23,6 +23,42 @@ impl LocalOrAws {
             Ok(LocalOrAws::Aws(signer))
         }
     }
+
+    /// Check if the key is present in the environment.
+    /// If so, load the signer (local privkey or AWS signer).
+    /// If not, return None.
+    pub async fn load_option(
```

Comment:
if we separate config loading from signer instantiation, we can delete all env access from this file, which feels a lot cleaner

Reply:
yeah, that's much much nicer: https://github.com/init4tech/zenith-rs/pull/12/commits/45aafb676d943c14654c788f5be7becb0b873694